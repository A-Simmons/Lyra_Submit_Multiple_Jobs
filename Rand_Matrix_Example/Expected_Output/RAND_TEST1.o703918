WARNING: ignoring environment value of R_HOME

-----
PBS Job 703918.pbs
CPU time  : 00:00:00
Wall time : 00:00:03
Mem usage : 25900kb
