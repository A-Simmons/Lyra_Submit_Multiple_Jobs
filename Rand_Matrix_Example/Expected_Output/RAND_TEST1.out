
R version 3.2.4 (2016-03-10) -- "Very Secure Dishes"
Copyright (C) 2016 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> args<-commandArgs(TRUE)
> 
> # Set some defaults
> seed <- 1
> rows <- 10
> cols <- 10
> 
> # Replace defaults with arguments if they exist
> for(i in 1:length(args)){
+   eval(parse(text=args[[i]]))
+ }
> 
> set.seed(seed)
> print(c(seed, rows, cols))
[1]  1 10 10
> print(matrix(rexp(200, rate=.1),rows,cols))
           [,1]      [,2]       [,3]       [,4]      [,5]      [,6]       [,7]
 [1,]  7.551818 13.907351 23.6451525 14.3528534 10.798811  4.222424  0.8967408
 [2,] 11.816428  7.620299  6.4189259  0.3726853 10.282469 21.787726 11.0817666
 [3,]  1.457067 12.376036  2.9412039  3.2401015 12.922616 32.177890  2.4726425
 [4,]  1.397953 44.239342  5.6586552 13.2046793 12.531054  5.578294 15.7198685
 [5,]  4.360686 10.545432  1.0607262  2.0351035  5.546414  5.946177 48.3281274
 [6,] 28.949685 10.352439  0.5943916 10.2272588  3.012830  9.773958  4.3113213
 [7,] 12.295621 18.760352  5.7871246  3.0174093 12.931247  2.098666 27.3038931
 [8,]  5.396828  6.547466 39.5893285  7.2521430  9.945558  3.094479 11.3683142
 [9,]  9.565675  3.369335 11.7331211  7.5154269  5.141743 11.059363  8.1336825
[10,]  1.470460  5.884797  9.9681296  2.3502745 20.078324  7.741878  8.3700649
            [,8]       [,9]     [,10]
 [1,] 17.8476540  3.8619356 10.350971
 [2,] 23.1247163 10.0825646  4.474519
 [3,] 29.0988727  8.1851419 10.436085
 [4,]  2.8559098  0.5926121  2.608282
 [5,]  3.8878677 22.8385347  6.812291
 [6,]  0.5205545  8.0417091  2.637383
 [7,]  3.5187050 15.8369608  4.466057
 [8,] 15.6524135 12.3379151  2.106069
 [9,]  8.1453581 13.4564402  1.325714
[10,] 27.5924379 21.0037723  3.488883
> 
> proc.time()
   user  system elapsed 
  0.180   0.032   0.248 
